redpanda/values.yaml

```diff
@@ -60,8 +61,26 @@ config:
 
     # rpc server listener
     rpc_server:
+      address: 0.0.0.0
       port: 33145
 
+  # panda proxy
+  pandaproxy:
+    enabled: true
+    pandaproxy_api:
+    - address: 0.0.0.0
+      name: internal
+      port: 8082
+
+  schema_registry:
+    enabled: true
+    schema_registry_api:
+    - address: "0.0.0.0"
+      name: internal
+      port: 8081
+
+  schema_registry_replication_factor: 1
```

Comment:
This doesn't look right.

Reply:
Compared it to https://doc.redpanda.com/docs./cluster-administration/configuration/ and looks like the indentation.